_non-cosmos/solana/assetlist.json

```diff
@@ -229,6 +229,301 @@
+      "type_asset": "erc20",
+      "address": "jupSoLaHXQiZZTSfEWMTRRgpnyFm8f6sZdosWBjx93v",
+      "denom_units": [
+        {
+          "denom": "jupSoLaHXQiZZTSfEWMTRRgpnyFm8f6sZdosWBjx93v",
+          "exponent": 0
+        },
+        {
+          "denom": "jupsol",
+          "exponent": 9
+        }
+      ],
+      "base": "jupSoLaHXQiZZTSfEWMTRRgpnyFm8f6sZdosWBjx93v",
+      "name": "Jupiter Staked SOL",
+      "display": "jupsol",
+      "symbol": "jupSOL",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/solana/images/jupsol.png"
+      },
+      "coingecko_id": "jupiter-staked-sol",
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/solana/images/jupsol.png"
+        }
+      ],
+      "socials": {
+        "website": "https://jup.ag/",
+        "twitter": "https://twitter.com/JupiterExchange"
+      }
+    },
+    {
+      "description": "dogwifhat is a meme coin that operates on Solana.",
+      "type_asset": "erc20",
+      "address": "EKpQGSJtjMFqKZ9KQanSqYXRcF8fBopzLHYxdM65zcjm",
+      "denom_units": [
+        {
+          "denom": "EKpQGSJtjMFqKZ9KQanSqYXRcF8fBopzLHYxdM65zcjm",
+          "exponent": 0
+        },
+        {
+          "denom": "wif",
+          "exponent": 6
+        }
+      ],
+      "base": "EKpQGSJtjMFqKZ9KQanSqYXRcF8fBopzLHYxdM65zcjm",
+      "name": "dogwifhat",
+      "display": "wif",
+      "symbol": "WIF",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/solana/images/wif.png"
+      },
+      "coingecko_id": "dogwifcoin",
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/solana/images/wif.png"
+        }
+      ],
+      "socials": {
+        "website": "https://dogwifcoin.org/",
+        "twitter": "https://twitter.com/dogwifcoin"
+      }
+    },
+    {
+      "description": "Tether, issued natively on Solana.",
+      "extended_description": "Tether USD (Native Solana), USDT is a native asset to Solana issued by Tether.",
+      "type_asset": "erc20",
+      "address": "Es9vMFrzaCERmJfrF4H2FYD4KCoNkY11McCe8BenwNYB",
+      "denom_units": [
+        {
+          "denom": "Es9vMFrzaCERmJfrF4H2FYD4KCoNkY11McCe8BenwNYB",
+          "exponent": 0
+        },
+        {
+          "denom": "usdt",
+          "exponent": 6
+        }
+      ],
+      "base": "Es9vMFrzaCERmJfrF4H2FYD4KCoNkY11McCe8BenwNYB",
+      "name": "Tether",
+      "display": "usdt",
+      "symbol": "USDT",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/solana/images/usdt.png"
+      },
+      "coingecko_id": "tether",
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/solana/images/usdt.png"
```

Comment:
this is a duplication--there are already USDT images in the ethereum/images/ directory. Please remove the new USDT.png here, and point to ethereum's USDT instead using image_sync

Reply:
updated